keanu-project/src/test/java/io/improbable/keanu/algorithms/mcmc/nuts/LeapfrogTest.java

```diff
@@ -71,8 +71,8 @@ public void canLeapForward() {
         Leapfrog start = new Leapfrog(position, momentum, gradient);
         Leapfrog leap = start.step(vertices, mockedGradientCalculator, EPSILON);
 
-        Assert.assertEquals(1.0, leap.getPosition().get(aID).scalar(), 1e-6);
-        Assert.assertEquals(1.0, leap.getPosition().get(bID).scalar(), 1e-6);
+        Assert.assertEquals(2.0, leap.getPosition().get(aID).scalar(), 1e-6);
```

Comment:
Leaping correctly means the position should progress to 1*gradient not 0.5*gradient.